Inspicio/Controllers/ImagesController.cs

```diff
@@ -288,15 +288,25 @@ public JsonResult GetRating(int? id)
         }
         public JsonResult GetComments(int? Id)
         {
-            List<CommentInfo> comments = new List<CommentInfo>();
-            var AllComments = _context.Comments.Where(c => c.ScreenId == Id);
+            List<CommentsViewModel> comments = new List<CommentsViewModel>();
+            var AllComments = _context.Comments.Where(c => c.ScreenId == Id).Include(x => x.ApplicationUser).ThenInclude(y => y.Comments);

```

Comment:
The `Include(x => x.ApplicationUser).ThenInclude(y => y.Comments)` part probably isn't needed, it looks like it's only `ApplicationUser` that's used.